husky-communication/husky-ppq/src/main/java/org/projecthusky/communication/ch/ppq/impl/clients/SimplePpfClient.java

```diff
@@ -26,43 +27,34 @@
 /**
  * Class implementing the simple ppq client.
  */
-public class SimplePpfClient extends CamelService
-		implements PpfClient {
+public class SimplePpfClient extends CamelService implements PpfClient {
```

Comment:
is there a known formatter class which can be used by all devs? If yes, please send me the config. Because I see differences even between the classes.

Reply:
Nothing as much widespread as formatters for JS, unfortunately. I agree it would be a nice thing to have.